react-backend/shared/models/cb-admin.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  getCbFeedback: async (dbConnection, cbId) => {
```

Comment:
If you imagine that this object is describing the `CbAdmin` model, then putting `CB` in the method name seems redundant; there are no other kinds of feedback we can collect.

```js
const feedback = await CbAdmin.getFeedback(client, id);

// compared to ...

const feedback = await CbAdmin.getCbFeedback(client, id);
```

Separate note (with no action right now): I usually like to make these methods as general as possible to capture future use cases more easily. For example, at the moment, this query can only give us all the feedback for a CbAdmin. But what if we want to filter by date in future? Wouldn't it be nice to be able to do something like:

```js
const then = new Date(Date.now() - 1e10)
const now = new Date(Date.now())
const feedback = await CbAdmin.getFeedback(client, { id, since: then, until: now });
```

Reply:
Oooooo